.circleci/test_migrations_from_6_0.sh

```diff
@@ -107,7 +107,7 @@ docker run --user www-data --rm \
   --volume $(pwd):/srv/pim --volume ~/.composer:/var/www.composer --volume ~/.ssh:/var/www/.ssh \
   --workdir /srv/pim \
   --env COMPOSER_AUTH \
-  akeneo/pim-php-dev:master \
+  akeneo/pim-php-dev:8.1 \
```

Comment:
We don't keep 8.0 for 6.0 to 7.0 migration test?

Reply:
The PHP8.0 was used here => https://github.com/akeneo/pim-community-standard/pull/1445/files#diff-baa0c3bbd53b2d3a29c95d798d2fcdf650704646ba56060b23a09c6a18929a6dR77.

Before the version was the PIM version now the version is the PHP version